templates/deploy_fortigate_autoscale.hybrid_licensing.json

```
@@ -314,6 +314,22 @@
                 "description": "Name of subnet 4, a subnet that will gain access from the FortiGate. When creating a new VNet that contains the 'Subnet 4', the input value will be used as the Subnet 4 name, or if left empty, a default name will be used. When using an existing Net, a valid non-empty input will assume the association of the target subnet with the Autoscale, and the target subnet will be associated as 'Subnet 4'; empty value will assume the Autoscale will deploy without using the 'Subnet 4'."
             }
         },
+        "TerminateUnhealthyVM": {
+            "defaultValue": "no",
+            "allowedValues": ["no", "yes"],
+            "type": "String",
+            "metadata": {
+                "description": "Default 'no'. When Set to 'no' VM termination and health will be determined by the Azure health probe and AutoHealing. This is the recommended value. When set to true, the Azure Function code will attempt to determine the health of the VM and will delete unhealthy VMs. "
```

The description is different from the AWS project: https://github.com/fortinet/fortigate-autoscale-aws/pull/69/files#diff-f29189982c71c174a00ebb10838d0fcbfa1a80e9826339eeb84176481a33494cR494

"VM termination and health will be determined by the Azure health probe and AutoHealing" needs to remove because the statement isn't true to this project. We do not add custom configuration in the project to terminate any vm by Azure health probe. Neither Azure does so by default. And AutoHealing is a feature of Azure web app. It won't trigger any vm termination either.

```suggestion
                "description": "Terminate any VM that is deemed unhealthy by the Autoscale. Default 'no'."
```

Can use exactly the same description as the AWS one because the descriptions there have been reviewed and approved by Kimberly. We can reuse it.

---

"Terminate any VM that is deemed unhealthy by the Autoscale. Default 'no'." Makes it sound like that is being determined by autohealing, Which is why I am hesitant to use it. 

Is there no health-probe setup in this project?

---

There are. There are load balancer setups that require probe but the probes only determine traffic flow not terminating vm.

---

```suggestion
                "description": "Terminate any VM that is deemed unhealthy by FortiGate Autoscale. The default is 'no'."
```

---

@Joel-Cripps  Does adding the word fotiGate will make it sound like autohealing?

---

@klc178 Please also revise it against the description here in the latest release (on AWS): https://github.com/fortinet/fortigate-autoscale-aws/blob/3.4.0/templates/autoscale-new-vpc.template.yaml#L503

This feature is an Autoscale feature generally available for all platforms. We should keep the consistent description for this parameter so let me know if the description needs to update on the AWS project too?